pkgs/development/libraries/science/astronomy/indilib/indi-firmware.nix

```diff
@@ -45,9 +48,33 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
-    for f in libfishcamp/CMakeLists.txt libsbig/CMakeLists.txt
+    for f in \
+     indi-qsi/CMakeLists.txt \
+     indi-dsi/CMakeLists.txt \
+     indi-armadillo-platypus/CMakeLists.txt \
+     indi-orion-ssg3/CMakeLists.txt \
+     libqhy/CMakeLists.txt \
+     libfishcamp/CMakeLists.txt \
+     libsbig/CMakeLists.txt \
```

Comment:
This can probably be changed to `*/CMakeLists.txt` without issue -- was there a reason you didn't do this?

Reply:
Not really, just was following the existing convention. I've made this change.